tensorboard/components/tf_storage/storage.ts

```diff
@@ -54,6 +54,11 @@ export let TAB = '__tab__';
 export let DISAMBIGUATOR = 'disambiguator';
 
 /**
+ * The (string) name for the run of the selected dataset in the graph dashboard.
+ */
+export const RUN = 'run';
```

Comment:
This constant shouldn't be in `storage.ts`; there's no reason that the core storage module should need to know about the graph dashboard. Instead, how about `const RUN_STORAGE_KEY = 'RUN';` in `tf-graph-dashboard.html`? (No need to export anything; it can be local within the `<script>`.)

Reply:
Done.